### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my quiz app homepage?

I have added a simple toggle feature to filter between different categories

Live Link : https://water-quiz-6z4lupncy-shlok30.vercel.app/

Please let me know if changes are required.

Thanks

### CODE REVIEW

File: src/components/quiz-card.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom"
+
+const QuizCard = ({title,id,category}) => {
+
+    return(
+        <div class="card-container flex flex-column gap-m" style = {{width : "30%"}}>
```
Comment (sharmayatendra) [inline src/components/quiz-card.js:6]: avoid inline styles

**Review by sharmayatendra — APPROVED** [review]
nothing major to change 
good work 
code is clean
good variable names used

File: src/context/quiz-context.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, useState } from "react";
+import { quizzes } from "../db/quizes";
+
+const QuizContext = createContext()
+
+const QuizContextProvider = ({children}) => {
+
+    const [quizDB, setQuizDB] = useState([...quizzes])
```
Comment (vishalg8454) [inline src/context/quiz-context.js:8]: can simplify with `useState(quizzes)`.

**Review by vishalg8454 — APPROVED** [review]
Simple to understand the code. Can merge.

**Pull request merged**